### DISCUSSION TITLE
Loki: Derived fields supporting multiple labels

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#62660] [Observability]
This feature request came from https://github.com/grafana/loki/issues/3119#issuecomment-1408674910.

I would love to be able to make custom buttons in Loki using label-populated regexes just like the buttons that appear to take the user to Tempo. My first use-case would be to use logged snowflake QueryIds to link to the query detail

ex:
https://app.snowflake.com/{{.snowflake_account}}/#/compute/history/queries/{{.query_id}}/detail

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82817. Feel free to continue the discussion around this there. Thank you!
